### DISCUSSION TITLE
AndExpand option alternative in MAUI Controls

### DISCUSSION THREAD

**Discussion by IXChinmayD** [dotnet/maui#21696] [Upgrading from Xamarin]
In Xamarin forms, **AndExpand** was making the Widget like entry, image, etc to acquire available width/height for itself. But, according to this article https://learn.microsoft.com/en-us/dotnet/maui/migration/layouts?view=net-maui-8.0, the **AndExpand** is absolute for MAUI controls. 
What is the new alternative solution?

**Comment by prabhavmehra** [comment]
Fill should do the job. Or you can use Grid with "*" in some cases where the grid takes the entire screen dimensions and fills the space as configured

**Reply by IXChinmayD** [reply]
Thank you @prabhavmehra for responding to my query, I will check.
